### DISCUSSION TITLE
Verbindet sich nicht mit WLAN Heimnetzwerk

### DISCUSSION THREAD

**Discussion by UeEiFetischist** [tbnobody/OpenDTU#982] [Q&A]
Guten Tag zusammen,

ich habe das Problem dass sich mein ESP32 mit CH340C Chip sich nicht mit mein Heimnetzwerk per WLAN verbinden möchte.
Mal geht es, meist aber nicht.
Kann es womöglich an meine WLAN SSID liegen, die Leerzeichen und ein Ausrufezeichen enthält? Aber alle anderen WLAN Geräte haben damit kein Problem...

Danke schon mal!

**Comment by Omega13x** [comment]
Hast wahrscheinlich eine "FRITZ!Box". Nein, daran liegt es nicht. Es liegt entweder an der Entfernung, oder an einem anderen WLan-Gerät zw. Router und ESP32. Ich vermute das du auch einen NRF24 angeschlossen hast. Wenn da die Sendeleistung zu hoch eingestellt ist, kann es auch zu Problemen kommen. Der ESP und der NRF senden und empfangen beide im 2,4Mhz-Bereich. Da musst du mit dem Standort etwas experimentieren. Auch das Abfrageintervall von der DTU und von MQTT sollten nicht gleich sein. Ich habe zum Beispiel 14Sek. - DTU und 5Sek -MQTT. So kollidiert die Abfrage nur alle 70 Sekunden.
